### PR TITLE
fix(@desktop/chat): fix various expression bugs in chat messages

### PR DESCRIPTION
**Pull request by PascalPrecht**
There are a bunch of references and expressions that are invalid.
This commit fixes them.

### CODE REVIEW

File: ui/app/AppLayouts/Chat/ChatColumn/ChatMessages.qml [hunk 1]
```diff
@@ -118,16 +118,16 @@ SplitView {
                     radius: 16
                 }
                 onClicked: {
-                    root.newMessages = 0
+                    svRoot.newMessages = 0
```
Comment (pinebit) [inline ui/app/AppLayouts/Chat/ChatColumn/ChatMessages.qml:121]: I fixed these in my today's PR...

Comment (PascalPrecht) [reply ui/app/AppLayouts/Chat/ChatColumn/ChatMessages.qml:121]: OH! Then we can ignore this one :D

File: ui/app/AppLayouts/Chat/ChatColumn/Message.qml [hunk 2]
```diff
@@ -305,7 +305,7 @@ Item {
             visible: {
                 return gapNowAndOldest < maxGapInSeconds
                         && gapNowAndJoined > maxGapInSeconds
-                        && (chatsModel.activeChannel.chatType !== Constants.chatTypePrivateGroupChat || chatsModel.activeChannel.isMember)
+                        && (chatsModel.channelView.activeChannel.chatType !== Constants.chatTypePrivateGroupChat || chatsModel.channelView.activeChannel.isMember)
```
Comment (pinebit) [inline ui/app/AppLayouts/Chat/ChatColumn/Message.qml:308]: this can in fact fix the new bug about "load more messages" invisible...
thanks!!!

**Review by pinebit — APPROVED** [review]

File: ui/app/AppLayouts/Chat/ChatColumn/ChatMessages.qml (hunk 1 above)
Comment (pinebit) [reply ui/app/AppLayouts/Chat/ChatColumn/ChatMessages.qml:121]: No! mine is not merged, so if yours comes first, then I will discard mine.

Comment (pinebit) [reply ui/app/AppLayouts/Chat/ChatColumn/ChatMessages.qml:121]: Actually let me cancel mine right now

Comment (pinebit) [reply ui/app/AppLayouts/Chat/ChatColumn/ChatMessages.qml:121]: Closed https://github.com/status-im/status-desktop/pull/2860

**Comment by pinebit** [comment]
Closes #2859.

**Comment by pinebit** [comment]
I believe this should fix https://github.com/status-im/status-desktop/issues/2856

**Review by jrainville — APPROVED** [review]

**Pull request merged**